packages/core/src/lib/label.svelte

```diff
@@ -50,7 +50,7 @@ export { extraClasses as cx };
   )}
 >
   <span
-    class={cx('flex text-xs', {
+    class={cx('text-xs', {
       'inline whitespace-nowrap': position === 'left',
```

Comment:
Oh yeah, flex was definitely a conflicting class with inline, and given the parent is flex, I don't really see why this inline is here either